packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -184,6 +186,9 @@ class _DismissibleState extends State<Dismissible> with TickerProviderStateMixin
   Size _sizePriorToCollapse;
 
   @override
+  bool get wantKeepAlive => _moveController?.isAnimating == true || _resizeController?.isAnimating == true;
```

Comment:
I'm gonna pass on this for now. We only have three clients, and they're all a bit different from each other currently. If patterns emerge then we can address them subsequently.